### PR TITLE
Refactor jm fix

### PR DESCRIPTION
**Pull request by milanmajchrak**
| Phases            | MP | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |  0  |    0 |     0 |      0 |        0 |
| Developing      |  0  |  1  |    0 |    0 |      0 |         0 |
| Review             |  0  |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -  |   -   |   -     |        0 |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ because we use this endpoint for importing existing data.
 > - `psql -U postgres clarin-utilities < <UTILITIES_DUMP_FILE_PATH>`
 
 or use `scripts/start.local.dspace.db.bat`.
+NOTE: `start.local.dspace.db.bat` and `init.dspacedb5.sh` must have `LF` line separator.
```
Comment (vidiecan) [inline README.md:39]: nope, bat must have CRLF
sh must have LF
ensure this using
https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings#per-repository-settings

```
*.sh eol=lf
*.py eol=lf
*.md eol=lf
*.bat eol=crlf
```

File: scripts/init.dspacedb5.sh [hunk 2]
```diff
@@ -8,11 +8,11 @@ createuser --username=postgres dspace
 
 echo "Importing clarin-dspace"
 createdb --username=postgres --owner=dspace --encoding=UNICODE clarin-dspace
-psql -U postgres clarin-dspace < ../dump/clarin-dspace-8.8.23.sql &> /dev/null
```
Comment (vidiecan) [inline scripts/init.dspacedb5.sh:11]: you sure we need this?

**Review by vidiecan — CHANGES_REQUESTED** [review]

File: scripts/init.dspacedb5.sh (hunk 2 above)
Comment (milanmajchrak) [reply scripts/init.dspacedb5.sh:11]: It is very helpful. Without it you must find out that imports were successful with logs that tell you they weren't.

Comment (vidiecan) [reply scripts/init.dspacedb5.sh:11]: this is dspace v5, so no import is being done into this database

Comment (milanmajchrak) [reply scripts/init.dspacedb5.sh:11]: I mean importing the dump into database

**Pull request merged**